tests/testthat/test-integration-logisticRegression.R

```diff
@@ -0,0 +1,67 @@
+context("Logistic Regression: helper functions testing ")
+
+config <- list(
+  `graph.resolution` = '1x',
+  `the.link` = 'logit',
+  `model.name` = 'Basic_Logistic_Regression',
+  `used.weights` = FALSE,
+  `X Vars` = c('gre', 'gpa', 'rank'),
+  `Y Var` = 'admit'
+)
+
+##----
+
+#' #### Read Inputs
+#'
+#' This is a named list of all inputs that stream into the R tool.
+#' We also specify defaults for use when R code is run outside Alteryx.
+defaults <- list(
+  data = admission[, c(config$`Y Var`, config$`X Vars`)]
+)
+inputs <- list(
+  the.data = read.Alteryx2("#1", default = defaults$data),
+  XDFInfo = getXdfProperties("#1", default = list(is_XDF = FALSE, xdf_path = NULL))
+)
```

Comment:
So, this can simply be

```r
inputs <- list(
  the.data = admission[, c(config$`Y Var`, config$`X Vars`)],
  XDFInfo = list(is_XDF = FALSE, xdf_path = NULL)
)
```

These tests are meant to be run on the R console, so there is no need to add the Alteryx related calls. This should let easier definition of tests.